deploy/web/landingapp/src/pages/LandingPage/LoginDisplay/index.js

```diff
@@ -0,0 +1,35 @@
+/*****
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+
+import "./styles.css";
+
+const LoginDisplay = ({ loginStepIncreaseHandler }) => {
+  /*--------------- HANDLERS ----------------*/
+  const loginHandler = (e) => {
+    e.preventDefault();
+    loginStepIncreaseHandler();
+  };
+  return (
+    <div>
+      <div style={{ justifyContent: "center", marginTop: "8px" }}>
+        <button className="login-form__submit" onClick={loginHandler}>
+          Sign In With Facebook
+        </button>
```

Comment:
As noted in the [design doc](https://docs.google.com/document/d/1phign6jexV0MHv-tUngZJd2FIjrJ6jxIXYg2djR5Coo/edit), we'll need a single checkbox to affirm before enabling this.